### ISSUE TITLE
[BUG] TimeSeriesForestClassifier not working with multiple outputs.

### ISSUE THREAD

**Issue by Stevellen** [alan-turing-institute/sktime#460]
**Describe the bug**

The TimeSeriesForestClassifier (TSFC) documentation suggests that the model should work with a multilabel/multioutput problem. When I try to run it this way, however, the training process seems to complete fine, but at inference time I get a ValueError from numpy saying that it cannot broadcast size (N,1) to (N,). For example, when I try to predict on a set with 8 possible classes (with the classes for each sample represented as a multi-hot vector) I get an error.

**To Reproduce**
```python

# these sets have shapes (22317, 4)  (7440, 4) (22317, 8)  (7440, 8)
X_tr, X_ts, y_tr, y_ts = train_test_split(data, labels)

pipe = Pipeline([
    ('concat', ColumnConcatenator()),
    ('classify', TimeSeriesForestClassifier(n_estimators=100, n_jobs=n_jobs))
    ])
pipe.fit(X_tr, y_tr.values)
result = pipe.score(X_ts, y_ts.values) # this line is the problem
```

**Expected behavior**
The model to output a score in this case, or a set of probability vectors when ```python predict_proba() ``` is called.

**Additional context**
Ultimately I am trying to solve a multitask problem here. Any help is very much appreciated.

**Versions**
<details>
System:
    python: 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 23:03:10) [MSC v.1916 64 bit (AMD64)]
executable: c:\users\steven.belcher\dev\redberry\snorkel_env\scripts\python.exe
   machine: Windows-10-10.0.18362-SP0

Python dependencies:
          pip: 20.2.4
   setuptools: 41.2.0
      sklearn: 0.23.2
        numpy: 1.18.5
        scipy: 1.5.3
       Cython: 0.29.21
       pandas: 1.1.3
   matplotlib: 3.3.2
       joblib: 0.17.0
        numba: 0.51.2
     pmdarima: None
      tsfresh: 0.17.0
</details>

**Comment by mloning** [comment]
Thanks @Stevellen for opening the issue, it's possible that this is a bug, I don't think we have a test for it and may have broken the support for multilabel classification when extending sklearn. 

Do you have some toy data to work with? Would you be interested in looking into this?

**Comment by Stevellen** [comment]
@mloning Sure, I could have a look at it sometime this weekend.

**Comment by nakamarusun** [comment]
Have the same issue. fit() function does not throw any errors. However, calling .predict() function returns a single array of shape (N,), where N is the number of labels, instead of (m, N), where m is the amount of data being predicted.

**Comment by nakamarusun** [comment]
Since the .predict() only returns a single array at a time, my workaround at the moment is inserting my data into the predict() function one by one in a for loop and count accuracy manually. So, my code to calculate the accuracy is something like:

```py
# X_te is X test data, and Y_te is Y test data

total = 0
for i in range(len(X_te)):
  pred = model2.predict(np.array((X_te[i], )))
  total += 1 if accuracy_score(pred, Y_te[i]) == 1.0 else 0
  
print(total/len(X_te))
```

**Comment by TonyBagnall** [comment]
if I understand it, you are trying to do multi-class classification, i.e. predict multiple target variables with a single classifier? If so, this capability is not (yet?) supported by sktime. We should throw a more meaningful error if it is attempted though

**Comment by TonyBagnall** [comment]
hi, in the refactor and error is thrown if y is not one dimensional. There are no medium term plans to make classifiers compatible for multi-label classification, so given little activity  on this, I will close this issue
